documentation/modules/ROOT/pages/connectors/oracle.adoc

```diff
@@ -4210,6 +4209,275 @@ After the connector's commit SCN advances beyond the DDL change, to prevent unpa
+  GRANT SELECT, FLASHBACK ON SYS.TAB$ TO c##dbzuser;
+  GRANT SELECT, FLASHBACK ON SYS.TABCOMPART$ TO c##dbzuser;
+  GRANT SELECT, FLASHBACK ON SYS.TABPART$ TO c##dbzuser;
+  GRANT SELECT, FLASHBACK ON SYS.TABSUBPART$ TO c##dbzuser;
+  GRANT SELECT, FLASHBACK ON SYS.TS$ TO c##dbzuser;
+  GRANT SELECT, FLASHBACK ON SYS.USER$ TO c##dbzuser;
+
+  exit;
+EOF
+
+----
+
+[[selecting-the-adapter]]
+=== Configuring the OpenLogReplicator adapter
+
+By default, {prodname} uses Oracle LogMiner to ingest change events from Oracle.
+You can adjust the connector configuration to enable the connector to use OpenLogReplicator.
+
+The following configuration example adds the property `database.connection.adapter`, `openlogreplicator.source`, `openlogreplicator.host`, and `openlogreplicator.port` to enable the connector to use the OpenLogReplicator adapter.
+
+[source,json,indent=0]
+----
+{
+    "name": "inventory-connector",
+    "config": {
+        "connector.class" : "io.debezium.connector.oracle.OracleConnector",
+        "tasks.max" : "1",
+        "topic.prefix" : "server1",
+        "database.hostname" : "<oracle ip>",
+        "database.port" : "1521",
+        "database.user" : "c##dbzuser",
+        "database.password" : "dbz",
+        "database.dbname" : "ORCLCDB",
+        "database.pdb.name" : "ORCLPDB1",
+        "schema.history.internal.kafka.bootstrap.servers" : "kafka:9092",
+        "schema.history.internal.kafka.topic": "schema-changes.inventory",
+        "database.connection.adapter": "olr",
+        "openlogreplicator.source": "ORACLE",
+        "openlogreplicator.host": "<ip address or hostname of OpenLogReplicator>",
+        "openlogreplicator.port": "<port OpenLogReplicator is listening on>"
+    }
+}
+----
+
+[id="building-openlogreplicator"]
+=== Building OpenLogReplicator
+
+OpenLogReplicator does not distribute binaries, so you must build the third-party tool from code.
+It is written in C++; however, the author provides a container image to build the source for a variety of operating systems such as Debian, Centos, Fedora, etc.
+This https://github.com/bersler/OpenLogReplicator-docker[repository] provides details on how to build the tool using a container.
+
+[id="obtaining-oracle-jdbc-driver-openlogreplicator"]
+=== Obtaining the Oracle JDBC driver for OpenLogReplicator
+
+The {prodname} Oracle connector requires the Oracle JDBC driver to connect to Oracle databases.
+Please refer to the link:#obtaining-the-oracle-jdbc-driver[Obtaining the Oracle JDBC driver] as OpenLogReplicator has the exact same requirements as Oracle LogMiner.
+
+[[oracle-openlogreplicator-configuration]]
+=== OpenLogReplicator configuration
+
+OpenLogReplicator is a third-party standalone process that is responsible for reading the Oracle redo and archive logs.
+Unlike other {prodname} Oracle adapters, this means that there are two configurable components that must be configured independently of one another.
```

Comment:
```suggestion
Unlike other {prodname} Oracle adapters, OpenReplicator includes two separate components, which you must configure independently of each other.
```

Reply:
I'm confused by the reference to the _two configurable components_. Along with the configuration described in this section, I assume that the sentence refers to the configuration described in `selecting-the-adapter`, which, I think is really less about configuring the component and more about specifying that you want to use the OpenLogReplicator.

If that's correct, we should move this sentence to earlier in the doc (e.g., L4357?) and refactor the topic _Configuring the OpenLogReplicator adapter_ so that it includes the subtopics _Selecting the adapter_, and _Configuring the adapter_. 
If that's not correct, I'm not sure what the second component is.

Or maybe just remove that sentence?

Reply:
What I am referring to is that unlike other connectors where you focus on providing a Debezium connector config that is used by Kafka Connect, Debezium Server, or the Embedded Engine to run the connector, the user must also correctly configure and start the OpenLogReplicator process.  In other words, this additional configuration file must be created and set with a rigid set of values for the integration to work.  

I'm open to how you'd like best to address this clearly and concisely.

Reply:
Ok, I think that I get it.  Maybe I was just getting tripped up by the adapter terminology, which is new to me.  I also don't know much about Debezium server or the Embedded Engine. 
In any case, I think it might better if we just avoid making the negative comparison (_"Unlike ..."_), and simply say, "To use the {prodname} Oracle connector with the OpenReplicator adapter, you must provide a configuration for the adapter that is separate from the xref:oracle-adding-connector-configuration[configuration that you specify for the connector itself]."

We don't really have a separate Oracle connector configuration topic for downstream, adding the link won't work in that context, but we can either punt on that for now, or leave out the link.

Reply:
> I'll apply that edit to L4418 so you can either commit it, or push it back if you think that it still needs work.

That's fine, feel free to pull down this branch and push a commit to it if you want; entirely up to you.

> We don't really have a separate Oracle connector configuration topic for downstream, so adding the xref won't work in that context, so, for now, we can either punt on doing any conditionalization of the sentence, or leave out the link.

I think since OpenLogReplicator isn't going downstream anytime soon, we can leave it as is for now with the link and we can revisit this when we "downstreamify" this huge piece of work.